maint-lib/stagelib/envglobals.py

```diff
@@ -15,6 +15,7 @@
 # Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols
 REQUIRED_ENV_VARS = OrderedDict([
     ('CEPH_VERSION',      'Ceph named version being built (e.g., luminous, mimic)'),
+    ('CEPH_POINT_RELEASE', 'Points to specific version of ceph (e.g -12.2.0) or empty'),
```

Comment:
Please keep to the preexisting right-column alignment.

Reply:
Right-side column alignment already exists to improve code readability for maintainers. Breaking readability will make maintenance harder in the future.

Reply:
I'll move all the code to right then to avoid error 231